scrut_mission_node:
  ros__parameters: 
    timeout: 10
    max_steering_angle: 45.0
    steering_angle_velocity: 36.0
    max_speed: 5.0
    max_acceleration: 5.0
    jerk: 1.0